daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -836,10 +844,6 @@ private[lf] object Speedy {
         commitLocation: Option[Location] = None,
         limits: interpretation.Limits = interpretation.Limits.Lenient,
     ): Machine = {
-      val pkg2TxVersion =
-        compiledPackages.interface.packageLanguageVersion.andThen(
-          TransactionVersion.assignNodeVersion
-        )
```

Comment:
The original issue is there: `compiledPackages` is a mutable field and `pkg2TxVersion` should be keep in sync.
The problem was not very apparent before we implement interface, as the preprocessor was ensuring all the package were preloading before interpretation.